src/components/register-in-contest/register-project.jsx

```diff
@@ -28,12 +28,50 @@ export default React.createClass({
         technical_description: "",
         system_requirements: "",
         source_url: "",
-        homepage: ""
+        homepage: "",
+        open_source: "true",
+        closed_source_reason: ""
         /*eslint-enable */
-      }
+      },
+      wantsOpenSource: true
     };
   },
 
+  renderOpenSource() {
+    let openSource = null;
+    if (this.state.project.category !== "multimedia") {
+        openSource = <Input type="select"
+                            label="Ești de acord ca proiectul să fie public (open-source) ? *"
+                            value={this.state.project.open_source}
+                            onChange={this.onOpenSourceChange}>
+          <option value="true">Da</option>
+          <option value="false">Nu</option>
+        </Input>
+    }
+    return openSource;
+  },
+
+  renderWantsOpenSource() {
+    let wantsOpenSource = null;
+    if (this.state.project.category !== "multimedia") {
+      if (this.state.project.open_source === "true") {
+        wantsOpenSource = <Input type="url"
+                                 placeholder="http://..."
+                                 valueLink={this.deepLinkState(["project", "source_url"])}
+                                 label="Adresa surselor și a documentației *"
```

Comment:
`Adresa surselor și a documentației` îmi sună puțin ciudat.
Ce ziceți de `Link către surse și documentație` ?